tests/linearGaussian_npse_test.py

```diff
@@ -212,10 +206,18 @@ def npse_trained_model(sde_type, prior_type):
         if prior_type == "gaussian" or prior_type is None
         else None,
         "num_dim": num_dim,
+        "x_o": zeros(num_dim),
+        "sde_type": sde_type,
+        "prior_type": prior_type,
     }
 
 
 @pytest.mark.slow
+@pytest.mark.parametrize(
+    "npse_trained_model",
+    list(product(SDE_TYPES, PRIOR_TYPES)),
+    indirect=True,  # So pytest knows to pass to the fixture
+)
```

Comment:
will this be treated as one pytest test case? maybe there is way to make it more explicit such we have a different case for every combination. this would make it easier to see which part is failing.

Reply:
Each SDE_TYPE - PRIOR_TYPE is a separate test case